ui/src/literature/components/CiteModalAction.jsx

```diff
@@ -70,15 +74,17 @@ class CiteModalAction extends Component {
     let citeContent = this.citeContentCacheByFormat[format];
```

Comment:
I think we can remove this `cache` and rely on the browser http caching only, could you at least drop a `TODO:` here.